src/components/filters/FilterList/index.jsx

```diff
@@ -131,6 +135,8 @@ FilterList.defaultProps = {
   onAfterDrag: () => {},
   hideZero: true,
   tierAccessLimit: undefined,
+  lockedTooltipMessage: 'l',
```

Comment:
Why `'l'` and `'d'` instead of empty string like in `FilterSection/index.jsx`?

Reply:
ah thats some garbage when I was testing... 🤦‍♂  thanks for catching that!